server/services/messaging_service.go

```diff
@@ -303,7 +303,14 @@ func (s *messagingServerImpl) CreateBlurb(ctx context.Context, in *pb.CreateBlur
 	id := puid.Next()
 	name := fmt.Sprintf("%s/blurbs/%d", parent, id)
 	now := ptypes.TimestampNow()
-
+	if len(b.LegacyRoomId) >0 {
+		roomId := b.LegacyRoomId
+		name := fmt.Sprintf("%s/blurbs/%d.%s", parent, id, roomId)
```

Comment:
```suggestion
		name = fmt.Sprintf("%s/blurbs/%d.%s", parent, id, roomId)
```
In Go, `:=` declares a _new_ variable and assigns the value. We want to use just `=` so that we assign the new value to the already delcared `name` variable.

Declaring a variable and not using it is an compilation error in Go.

Reply:
👍